setup.py

```diff
@@ -14,6 +14,7 @@
     url="https://github.com/hectorespert/python-oligo",
     packages=setuptools.find_packages(),
     install_requires=[
+        'aiohttp',
```

Comment:
Could this dependency be optinal?

Reply:
As there is an import of both classes in the __init__.py file, there will be a crash because the lib is missing, even though you are not using the aio version.

You can try to capture the import's exception and raise a warning saying the lib is missing. But it's a bit confusing getting a warning when you are importing the sync version too.

In the end, it's your lib. What are you thoughts about it?

Reply:
I'm going to merge it in a development branch and try to find a solution for that.

Thanks @earada